lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -51,6 +51,13 @@ def authorize(money, payment, options = {}, capture = false)
         commit('payments', post)
       end
 
+      def capture(money, authorization, options = {})
+        payment = authorization.split('|').first
+        post = build_refund_request(money, options)
```

Comment:
let's rename `build_refund_request` to something more generic too prevent confusion - seems weird we are using it for captures. maybe something like `build_reference_request`

Reply:
you are right @curiousepic, change done